With 4.15+ kernels, Linux Intel SATA has a better link power saving policy,
med_power_with_dipm, which should be the recommended one to use
--- a/etc/laptop-mode/conf.d/intel-sata-powermgmt.conf
+++ b/etc/laptop-mode/conf.d/intel-sata-powermgmt.conf
@@ -34,6 +34,6 @@
 # while still trying to enable power-saving features), "max_performance" (power
 # management is disabled). Note that setting any level besides
 # "max_performance" also disables device hotplug.
-BATT_SATA_POLICY=min_power
+BATT_SATA_POLICY=med_power_with_dipm
 LM_AC_SATA_POLICY=max_performance
 NOLM_AC_SATA_POLICY=max_performance
